render.yaml

```diff
@@ -0,0 +1,10 @@
+services:
+  - type: web
+    name: MYAPPNAME # the name of this service, eg hello-world
```

Comment:
Do you think `MYAPPNAME` should be snake cased or separated by hyphens to improve readability?